DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreStringArray.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@model string[]
+@if (Model.Length == 1) {
```

Comment:
The reason that this partial determines behaviour based on the number of entries is this:
The OrganisationsCovered and TrainingVenues that use this partial to be displayed are stored in the database as a single string. They were populated on the old site by a textarea without any constraints on formatting, so users entered whatever they thought was sensible. 

This means you sometimes get multiple organisations/venues that are separated on new lines (which is dealt with when populating the view model) and sometimes you get multiple organisations/venues written as a sentence on a single line separated by commas ("and" or "&" sometimes separates values, there could be more that I haven't noticed). We can't separate the values apart based on these, as some single organisations/venues have commas etc. in their regular name or as part of an address.

As such, if we can separate the values based on new lines, then we shall display them on new lines in a list. If it is just a single value that we cannot separate, then we just display the text. If we displayed the single items in a list, then the screen reader will read "List with 1 item: [value]" which would be confusing if that value actually contained multiple organisations/venues separated by commas. So this logic makes sure no list is used if we can't separate the values.

Rather annoyingly, there are a few TrainingLocations that are given as an address, where each address line is separated onto a new line, and so will be displayed in a list. I think this is acceptable, as I think they are all single addresses.